extensions/notebook/src/jupyter/jupyterSessionManager.ts

```diff
@@ -343,11 +349,16 @@ export class JupyterSession implements nb.ISession {
 
 			let server = vscode.Uri.parse(utils.getLivyUrl(connectionProfile.options[KNOX_ENDPOINT_SERVER], connectionProfile.options[KNOX_ENDPOINT_PORT])).toString();
 			let doNotCallChangeEndpointParams: string;
+			let doNotCallChangeEndpointLogMessage: string;
 			if (utils.isIntegratedAuth(connectionProfile)) {
 				doNotCallChangeEndpointParams = `%_do_not_call_change_endpoint --server=${server} --auth=Kerberos`;
+				doNotCallChangeEndpointLogMessage = doNotCallChangeEndpointParams;
 			} else {
-				doNotCallChangeEndpointParams = `%_do_not_call_change_endpoint --username=${knoxUsername} --password=${knoxPassword} --server=${server} --auth=Basic_Access`;
+				doNotCallChangeEndpointParams = `%_do_not_call_change_endpoint --username=${knoxUsername} --server=${server} --auth=Basic_Access`;
+				doNotCallChangeEndpointLogMessage = doNotCallChangeEndpointParams + ` --password=${'*'.repeat(knoxPassword.length)}`;
```

Comment:
I wanted to have *something* to use for comparing whether the password was possibly correct - since the issue I'm trying to figure out is it returning a 401 so is some kind of auth error.

This isn't terribly useful and I'm going to get rid of it eventually, but for now I don't think there's any security concern here and it might prove slightly useful.